### ISSUE TITLE
The script detected nothing. 

### ISSUE THREAD

**Issue by ike20013** [theAIGuysCode/yolov4-deepsort#63]
Good afternoon, I did everything step by step and did not change anything in Google Colab through your notepad. But absolutely nothing was detected. I can't figure out what the problem is.
Thank you in advance for your response.
![Снимок экрана 2021-04-29 в 21 54 17](https://user-images.githubusercontent.com/61027857/116624586-dd49a100-a950-11eb-9937-c427028c16fe.png)
![Снимок экрана 2021-04-29 в 21 54 25](https://user-images.githubusercontent.com/61027857/116624591-e175be80-a950-11eb-8902-c3d653ac919d.png)

**Comment by jmudy** [comment]
What version of Tensorflow are you using? Try installing the requirements in a virtual environment. I had that problem because I was using a version of Tensorflow higher than `2.3.0rc0`. Let me know if the problem is solved!

**Comment by triethanol** [comment]
> 
> 
> What version of Tensorflow are you using? Try installing the requirements in a virtual environment. I had that problem because I was using a version of Tensorflow higher than `2.3.0rc0`. Let me know if the problem is solved!

Hi! I'm having the same problem. No bounding boxes are showing up and I am currently using tensorflow version 2.5.0-rc1.
How can I install the requirements in a virtual environment? I am still knew to programming in python and I have no idea. Please help ;-;

**Comment by jmudy** [comment]
I can see that you are using Google Colab. Try uncommenting the `#!pip install -r requirements-gpu.txt` cell and run it to install the libraries and dependencies.

**Comment by triethanol** [comment]
> I can see that you are using Google Colab. Try uncommenting the `#!pip install -r requirements-gpu.txt` cell and run it to install the libraries and dependencies.

No. I'm actually running this on my desktop. How do I create a virtual environment to run this in the recommended tensorflow version? I think this might be the issue.

**Comment by jmudy** [comment]
I recommend you to install Anaconda to create virtual environments.

**Comment by zhanghahah1993** [comment]
i also meet this problem.
i create new virtual environments and install requirment.txt,but still like that.
even it cant show the opencv's function.
pls help us.ty

**Comment by senemaktas** [comment]
I get same error, cannot see tracking .. When I run example video and code , I got that end of process -> **Video has ended or failed, try a different video format!**  Like ı said , I am using example videos , I have tried mine as well. Running on Colab. Have not changed any code. How can solve that? Thanks.

**Comment by PrakharPrashantSingh** [comment]
Hi, i am also facing same problem, where i m not getting any detections (bounding boxes), i tried latest Tensorflow GPU version i.e. 2.7.0 also tried afew old versions like 2.5 and 2.3 but still getting no detections.

Somebody please help me resolve it

Thanks

**Comment by dhruv101** [comment]
Try to use Tensorflow version 2.2.2. That worked for me.

**Comment by PHCLenzi** [comment]
> 2.2.2.

I had to uncomment this line as stated, but I also had to change the tensorflow version from 2.3.0rc0 to 2.3.0. In addition, I had to rotate this same cell twice. I'm running on colab and it worked.
